CONTRIBUTING.md

```diff
@@ -0,0 +1,216 @@
+
+Once you have set up GitHub to use SSH authentication, you should change the addresses of your Netpyne-lab GitHub remotes, from `https://` addresses to `git@` addresses, so that git knows to connect via SSH instead of HTTPS.
+>**Note:** You only could have a _SSH key_ from a repository you are a contributor.
+
+## Opening an issue
+
+You should usually open an issue in the following situations:
+
+* Report an error you can’t solve yourself.
+* Discuss a high-level topic or idea (for example, community, vision or policies)
+* Propose a new feature or other project idea
+
+Tips for communicating on issues:
+
+* **If you see an open issue that you want to tackle**, comment on the issue to let people know you’re on it. That way, people are less likely to duplicate your work.
+* **If an issue was opened a while ago**, it’s possible that it’s being addressed somewhere else, or has already been resolved, so comment to ask for confirmation before starting work.
+* **If you opened an issue, but figured out the answer later on your own**, comment on the issue to let people know, then close the issue. Even documenting that outcome is a contribution to the project.
+
+> **Note:** See in [GitHub Help](https://help.github.com/en) how to [creating an issue](https://help.github.com/en/articles/creating-an-issue).
+
+## Opening a pull request
+
+You should usually open a pull request in the following situations:
+
+* Submit trivial fixes (for example, a typo, a broken link or an obvious error).
+* Start work on a contribution that was already asked for, or that you’ve already discussed, in an issue.
+
+A pull request doesn’t have to represent finished work. It’s usually better to open a pull request early on, so others can watch or give feedback on your progress. Just mark it as a “WIP” (Work in Progress) in the subject line. You can always add more commits later.
+
+Always reference the issue related to the PR with `#number-of-issue` and use the markdown to make your comments more readable.
+
+> **Note:** If this is your first pull request, check out [Make a Pull Request](http://makeapullrequest.com/), which [**@kentcdodds**](https://github.com/kentcdodds) created as a walkthrough video tutorial. You can also practice making a pull request in the [First Contributions](https://github.com/Roshanjossey/first-contributions) repository, created by [**@Roshanjossey**](https://github.com/Roshanjossey).
+
+## Coding conventions
+
+**Adhere to standard Python style guidelines**
+All contributions to Netpyne are checked against style guidelines described in PEP 8. We also check for common coding errors (such as variables that are defined but never used).

```

Comment:
All contributions to NetPyNE are checked against style guidelines described in PEP 8 (we are gradually updating the existing code).

Reply:
:+1: I change for:
**Adhere to standard Python style guidelines**
All contributions to Netpyne are checked against style guidelines described in PEP 8 (we are gradually updating the existing code). We also check for common coding errors (such as variables that are defined but never used).

Reply:
NetPyNE